src/components/Modal/src/ModalArrow.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<span
+		:class="[
+			$s.ModalArrow,
+			$s[`position_${position}`],
+		]"
+		@click="$emit('click')"
+	>
+		<component
+			:is="icon"
+			:class="$s.ModalArrowIcon"
+		/>
+	</span>
+</template>
+
+<script>
+import ChevronLeft from '@square/maker-icons/ChevronLeft';
+import ChevronRight from '@square/maker-icons/ChevronRight';
+import modalApi from './modal-api';
+
+export default {
+	components: {
+		ChevronLeft,
+		ChevronRight,
+	},
+
+	inject: {
+		modalApi,
+	},
+
+	props: {
+		position: {
+			type: String,
+			default: 'left',
+			validator: (position) => ['left', 'right'].includes(position),
+		},
+	},
+
+	computed: {
+		icon() {
+			return `chevron-${this.position}`;
+		},
+	},
+};
+</script>
+
+<style module="$s">
+.ModalArrow {
+	--arrow-color: #fff;
+	--arrow-size: 24px;
+	--arrow-size-half: calc(var(--arrow-size) / 2);
+
+	position: absolute;
+	top: calc(50% - var(--arrow-size-half));
```

Comment:
if you have `top: calc(50% - var(--arrow-size-half))` here then I'm pretty sure you don't need the `transform: translateY(-50%)` below because all those do is move the arrow up half its height...